mautrix_hangouts/portal.py

```diff
@@ -575,7 +575,10 @@ async def handle_hangouts_message(self, source: 'u.User', sender: 'p.Puppet',
         if event.attachments:
             self.log.debug("Processing attachments.")
             self.log.trace("Attachments: %s", event.attachments)
-            event_id = await self.process_hangouts_attachments(event, intent)
+            try:
+                event_id = await self.process_hangouts_attachments(event, intent)
+            except:
```

Comment:
The errors should probably be logged even if they're ignored

```suggestion
            except Exception:
                self.log.warning("Failed to process attachment", exc_info=True)
```